### DISCUSSION TITLE
No Calendar & Kanban view

### DISCUSSION THREAD

**Discussion by idntty** [nocodb/nocodb#832] [Discord Threads]
There is no Calendar & Kanban view in last Docker or installation from source.

I tried installing the solution in different ways, tried different sources as databases - SQLLite or PG. But not the application does not allow to run these views.

**Comment by AztrexDX** [comment]
These are currently not present. May be, they are implementing this feature.

**Comment by bit-lang** [comment]
the same issue / question comes up several times. looking forward to it getting resolved.

**Comment by devsvc** [comment]
but they have this in the pic, and videos. feel being fooled after try.

**Reply by TechMasterElvy** [reply]
same here!

**Reply by BillMeyerRSA** [reply]
I thought I was missing something. Was about to try other types of install methods.

**Comment by TechMasterElvy** [comment]
They just released Calendar and Kanban view in latest version! 🌟

**Reply by o1lab** [reply]
Sorry, we've not!

**Comment by geosmart** [comment]
when does kanban and calender view come out

**Reply by matthewblott** [reply]
Kanban and Calendar views both have videos for the demos. It appears these have been unavailable for a long time and the maintainers have been aware. It's very misleading and likely to lead to a lot of people wasting time as these will be sought after features.

**Reply by wingkwong** [reply]
Some contents in main website are outdated and we're revamping the website. The outdated content will be removed in the next release. Sorry for any inconvenience caused.

**Reply by Myddleware** [reply]
Hi  @wingkwong ,
First of all, thanks for the great solution NocoDB. Well done !

Do you have any idea of ​​when the agenda view will be available again?
Thanks.

**Reply by endolith** [reply]
Kind of the opposite of "outdated" if you're showing stuff from the future that doesn't exist yet

**Comment by LeFanch** [comment]
Hello,
We wanted to run a POC with NocoDB then install a server for that.  But we didn't know that the calendar view was a "fake", and it was an important feature for us.
It's strange to see unavailable features on screenshots for an open source project, event with a tiny "coming soon" text.

Do you still plan to make one (and when) ?
Thank you.

**Comment by michacassola** [comment]
Any news on the ETA of Calendar and Kanban?

**Reply by StarkZarn** [reply]
Also looking for this :point_up:

Any news? The graphics are still up everywhere.

**Reply by wingkwong** [reply]
Kanban View will be available in next release. Calendar View is still in roadmap.

**Reply by strich** [reply]
Is there a roadmap publicly available anywhere?

**Reply by ivishalgandhi** [reply]
Any ETA for the calendar view ?
